test/Passes/DefaultPipeline/mixed.mlir

```diff
@@ -116,6 +91,8 @@ func.func @heap_to_stack(%A: memref<4x8xf32>,
   memref.copy %0, %C : memref<4x4xf32> to memref<4x4xf32>
 
   // CHECK-NOT: memref.dealloc
```

Comment:
I think we should remove this check?

Reply:
This one tests heap to stack conversion so it replaces alloc+dealloc with alloca.